src/wiki/plugins/metadata/forms.py

```diff
@@ -346,7 +280,7 @@ def __init__(self, article, request, *args, **kwargs):
         self.fields['slug'].widget = wiki.forms.TextInputPrepend(
             prepend='/' + self.article.urlpath_set.all()[0].path,
             attrs={
-                'pattern': UNICODE_LETTERS_HYPHEN_APPOS,
+                'pattern': UNICODE_LETTERS_MARKS_HYPHEN_APPOS,
```

Comment:
Add comment that this is client-side validation (also for supersense slug field)

Reply:
Also done!